pkg/nvml/mock/dgxb200/dgxb200.go

```diff
@@ -0,0 +1,473 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dgxb200
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/google/uuid"
+
+	"github.com/NVIDIA/go-nvml/pkg/nvml"
+	"github.com/NVIDIA/go-nvml/pkg/nvml/mock"
+)
+
+type Server struct {
+	mock.Interface
+	mock.ExtendedInterface
+	Devices           [8]nvml.Device
```

Comment:
Do the GB200 nodes not only have 4 devices? Or are these each addressable as 2?

Reply:
I think the difference here is that this is a B200 and not a GB200 -- the latter has 4.

Reply:
Ah yeah, but here (this PR) is a B200